### ISSUE TITLE
INSERT diagram isn't intuitive

### ISSUE THREAD

**Issue by jseldess** [cockroachdb/docs#5396]
On Slack, @darinpp mentioned:

> Hi Jesse. I was just looking at this https://www.cockroachlabs.com/docs/stable/insert.html and it doesn't seem to be showing the insert into ... values(...) syntax in the BNF diagram. Am I missing something? For Postgress for example - https://www.postgresql.org/docs/11/sql-insert.html - they show first the default values (which we have), then values (which I can't see) and then the query.

I think the diagram using `select_stmt` to represent various types of “selection clauses", including `VALUES`. The link to https://www.cockroachlabs.com/docs/stable/selection-queries.html in the `select_stmt` parameter description suggests as much. 

However, I very much agree with @darinpp that the diagram isn't friendly or intuitive. It should help users understand the statement syntax, as they most often use it, without having to dig deep and bounce around. We should revisit this and find a way to make the diagram express the most common cases, including `VALUES`. 

cc: @knz, who was involved in the way we documented "selection clauses". @rmloveland, @ericharmeling, not sure which of you wants to take this. Not sure if this is p1 or p2.

**Comment by knz** [comment]
It seems reasonable to me to present both the *general* form (the one currently presented) and then underneath in sub-sections expand the `select_stmt` clause into 3+ example diagrams including the one that mentions `VALUES`.

**Comment by ericharmeling** [comment]
I agree with @knz ... We do this for a few other SQL statements (e.g. [CREATE TABLE](https://www.cockroachlabs.com/docs/stable/create-table.html), with "Basic" and "Expanded" diagrams)

**Comment by darinpp** [comment]
The `VALUES` is not a general use select statement. It would make sense to have it expanded as part of defining the `INSERT`. The easiest to understand in my mind is the approach that MySQL has - https://dev.mysql.com/doc/refman/8.0/en/insert.html
It shows separately each of the variants.

**Comment by knz** [comment]
> The VALUES is not a general use select statement.

In fact, it is.

**Comment by darinpp** [comment]
You are right. I didn't know that. Everything is good then.

**Comment by jseldess** [comment]
Not sure if we'll have the bandwidth to update this for 19.2, so putting under 20.1.
